koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -424,6 +425,19 @@ def _write_logs(self, build_id, logs_dir, platforms: list = None):
                 task_platform = 'noarch'
 
             if task_platform not in logfiles:
+                if task_platform != 'noarch' and not final_platforms:
+                    final_platforms = self.osbs().get_final_platforms(build_id)
+
+                    if not final_platforms:
+                        self.logger.info("Couldn't obtain final platforms from build")
+                        final_platforms = platforms
+
+                    else:
+                        final_platforms = final_platforms
```

Comment:
what is the benefit of this assignment?

Reply:
will remove it, I was modifying that assigned value before